<launch>

	<!-- PID gains -->
	<arg name="p_gain" default="50"/>
	<arg name="i_gain" default="0"/>
	<arg name="d_gain" default="20"/>

	<!-- Experimental task (0,1,2) -->
	<arg name="task" default="0"/> 

	<!-- Method type ('A'- impedance, 'B'- learning) -->
	<arg name="methodType" default="A"/>

	<!-- velocity-based with trajopt planning PID -->
	<node name="pid_trajopt" pkg="iact_control" type="pid_trajopt.py" args="$(arg p_gain) $(arg i_gain) $(arg d_gain) $(arg task) $(arg methodType)" output="screen"/>

</launch>
